great_expectations/init_notebooks/pandas/validation_playground.ipynb

```diff
@@ -30,7 +30,9 @@
    "metadata": {},
    "source": [
     "## 1. Get a DataContext\n",
-    "This represents your **project** that you just created using `great_expectations init`. [Read more in the tutorial](https://docs.greatexpectations.io/en/latest/getting_started/create_expectations.html?utm_source=notebook&utm_medium=create_expectations#get-datacontext-object)"
+    "This represents your **project** that you just created using `great_expectations init`. \n",
+    "\n",
+    "[Read more in the tutorial](https://docs.greatexpectations.io/en/latest/tutorials/validate_data.html?utm_source=notebook&utm_medium=validate_data#get-a-datacontext-object)"
```

Comment:
I intentionally compacted these newlines to make the notebook less overwhelming. Shorter is better.

Reply:
+1